sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -164,7 +164,7 @@ static az_result append_simple_json_token(az_json_writer* jw, az_json_token* jso
  * temperature reached since boot.
  *
  * On initial bootup of the device, the sample will send the Temperature Controller reported
- * properties to the service. You will see the following in the device twin JSON.
+ * properties to the service. You will see the following in the device property JSON.
  *   {
```

Comment:
In general for documenting the JSON here (and even in API docs if it's still there?).  Is this really needed?  My gut is no.  If I'm a dev either I don't care and want the API to "just work" for me, or otherwise I can look at the .c implementing this or even better yet the PnP protocol spec someplace else.  This feels duplicative in this spot to me.

Reply:
I think I agree that the amount of doc's here is a little overwhelming. The one thing I will say is that because we are working at this low level (and the user as well), I'd argue it is useful to show the user what to expect since they will have to interact with this solution at a lower level than some other SDKs.